Makefile

```diff
@@ -4,7 +4,7 @@ test:
 
 .PHONY: lint
 lint:
-	golangci-lint run
+	golangci-lint run --config .golangci.yml
```

Comment:
[non blocking] I don't think we need to specify the configuration here as `golangci-lint` looks for it by default https://golangci-lint.run/usage/configuration/#config-file